semantic-conventions/src/tests/data/markdown/metrics_tables/expected.md

```diff
@@ -0,0 +1,15 @@
+# Test Markdown
+
+**`foo.size`**
+<!-- semconv metric.foo.size(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `foo.size` | Histogram | `{bars}` | Measures the size of foo. |
+<!-- endsemconv -->
+**Attributes for `foo.size`**
```

Comment:
_Maybe_ an empty line between metric table and attributes table makes the .md file easier to read/review?

Reply:
Done :)